service/controller/internal/templates/cloudformation/tccp/record_sets.go

```diff
@@ -67,6 +67,15 @@ const RecordSets = `
       Name: '{{ $v.EtcdDomain }}.'
       HostedZoneId: !Ref 'HostedZone'
       Type: A
+  EtcdNodeRecordSet:
+    Type: AWS::Route53::RecordSet
+    Properties:
+      ResourceRecords:
+      - !GetAtt {{ $v.MasterInstanceResourceName }}.PrivateIp
+      Name: 'etcd0.{{ $v.ClusterID }}.k8s.{{ $v.BaseDomain }}.'
+      HostedZoneId: !Ref 'InternalHostedZone'
+      Type: A
+      TTL: '60'
```

Comment:
Why a TTL?

Reply:
It's required if you specify `ResourceRecords`. AWS API rejects it otherwise. I took this from `aws-operator@master`.